### DISCUSSION TITLE
bo

### DISCUSSION THREAD

**Discussion by JESUS1959** [community/community#142583] [Accessibility]
### Select Topic Area

Question

### GitHub Feature Area

Discussions

### Body

ALGUIEN ME PUEDER AYUDAR?  LOS BOTONES DE MI GRABAADOR DEJARON DE FUNCIONAR, CUANDO LOS PULSO EL LASER NO SE ENCIENDE
![error grbl](https://github.com/user-attachments/assets/235d3129-3b82-4844-a661-373abc2b2f98)

**Comment by queenofcorgis** [comment]
Hi @JESUS1959 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on the tool or program you're using. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
